exporter/logzioexporter/config.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logzioexporter
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config contains Logz.io specific configuration such as Account Token, Region, etc.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"`
+	Token                         string `mapstructure:"account_token"`				// Your Logz.io Account Token, can be found at https://app.logz.io/#/dashboard/settings/general
+	Region                        string `mapstructure:"region"`					// Your Logz.io 2-letter region code, can be found at https://docs.logz.io/user-guide/accounts/account-region.html#available-regions
+	CustomListenerAddress         string `mapstructure:"custom_listener_address"`	// Custom endpoint to ship traces to. Use only for dev and tests.
```

Comment:
Exporters typically use a config option called `Endpoint` for this. It seems like the purpose is the same. For consistency it is best to use the same option name unless it is functionally different. I can't tell if it is different by just reading the comment since it doesn't tell what the expected format is.

Reply:
The address for the traces endpoint will be constructed using the region parameter. I will change to CustomEndpoint